lib/core/backup/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
I reckon it will be better for tests to create a new directory in `/tmp` so that they're less susceptible to differences in local state.